code/modules/mob/living/simple_animal/abnormality/teth/my_sweet_home.dm

```diff
@@ -0,0 +1,96 @@
+//Brain go brrrr.
+/mob/living/simple_animal/hostile/abnormality/my_sweet_home
+	name = "My Sweet Home"
+	desc = "This cozy little house is a safe nest built only for you. Everything is here for you..."
+	icon = '---'
+	icon_state = "---"
+	icon_living = "---"
+	icon_dead = "---"
+	threat_level = TETH_LEVEL
+	can_breach = TRUE
+	del_on_death = TRUE
+	var/icon_aggro = "---"
+	var/lc13admin_mode_variable = 0
+	max_boxes = 13
+	maxHealth = 600
+	health = 500
+	move_to_delay = 4
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 1, WHITE_DAMAGE = 1, BLACK_DAMAGE = 1, PALE_DAMAGE = 2)
+	melee_damage_lower = 5
+	melee_damage_upper = 5
+	melee_damage_type = RED_DAMAGE
+	armortype = RED_DAMAGE
+	rapid_melee = 1
+	melee_queue_distance = 3
+	retreat_distance = 2
+	minimum_distance = 1
+	stat_attack = CONSCIOUS
+	attack_verb_continuous = "stomps"
+	attack_verb_simple = "stomp"
+	deathmessage = "crumbles."
+	faction = list("hostile")
+	zone_selected = BODY_ZONE_CHEST
+	start_qliphoth = 1
+	vision_range = 10
+	aggro_vision_range = 20
+	wander = 1
```

Comment:
This is not needed, as it is inhereted already + abnos can patrol.

Reply:
Noted.